source/Octostache/Templates/Functions/TextValidationFunction.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Octostache.Templates.Functions
+{
+    internal static class TextValidationFunction
+    {
+        public static string? Match(string? argument, string[] options)
+        {
+            if (argument == null || options.Length != 1)
+                return null;
+
+            return Regex.Match(argument, options[0]).Success.ToString().ToLowerInvariant();
+        }
+        
+        public static string? StartsWith(string? argument, string[] options)
+        {
+            if (argument == null || options.Length != 1)
+                return null;
+
+            return argument.StartsWith(options[0]).ToString().ToLowerInvariant();
+        }
+        
+        public static string? EndsWith(string? argument, string[] options)
+        {
+            if (argument == null || options.Length != 1)
+                return null;
+
+            return argument.EndsWith(options[0]).ToString().ToLowerInvariant();
+        }
+        
+        public static string? Contains(string? argument, string[] options)
+        {
+            if (argument == null || options.Length != 1)
+                return null;
+
+            return argument.Contains(options[0]).ToString().ToLowerInvariant();
```

Comment:
This defaults to `Ordinal`. The others likely do, but lets be explicit. There is no StringComparison overload for `Contains` on .NET 4.0

Reply:
Thanks, wasn't aware of the overload. Checked the doc for StartsWith and looks like it doesn't use Ordinal so we need to specify it.
![image](https://user-images.githubusercontent.com/11970672/118066780-ad83aa00-b3e2-11eb-8788-597909a1cdf7.png)